src/main/java/com/github/onsdigital/JWTKeyProvider.java

```diff
@@ -0,0 +1,62 @@
+package com.github.onsdigital;
+
+import com.google.api.client.http.GenericUrl;
+import com.google.api.client.http.HttpBackOffUnsuccessfulResponseHandler;
+import com.google.api.client.http.HttpRequest;
+import com.google.api.client.http.HttpResponse;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.util.Data;
+import com.google.api.client.util.ExponentialBackOff;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class JWTKeyProvider {
```

Comment:
Can you create an interface for this functionality please i.e.:
```
public interface JWTKeyProvider{
    Map<String, String> getJwtKeys() throws Exception;
}
```
And rename and update this class to implement the interface:
```
public class JWTKeyProviderImpl implements JWTKeyProvider {
   // Existing code here.
}
```
Creating this abstraction helps to maintain a separation of concerns between components and prevents tight coupling, it also make testing easier as this can be easily subbed for a mock.